modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,17 @@
+data "aws_subnet" "vpc_private_subnet" {

```

Comment:
Why is this necessary? Why can't we use `var.private_subnets` directly?

Reply:
Yeah, I guess we can use it directly; https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest#output_private_subnets
Checking value passing in other ways is more interesting :-)